src/mantle/testing/factory/class-post-factory.php

```diff
@@ -46,46 +47,83 @@ public function __construct( Generator $generator, string $post_type = 'post' )
 		$this->post_type = $post_type;
 	}
 
+	/**
+	 * Create a new factory instance to create posts with a set of terms.
+	 *
+	 * @param array<int, WP_Term|int|string>|WP_Term|int|string ...$terms Terms to assign to the post.
```

Comment:
```suggestion
	 * @param array<int, WP_Term|int|string>|WP_Term|int|string> ...$terms Terms to assign to the post.
```
I think this is right? If not, ignore me